src/types/profile.js

```diff
@@ -518,6 +515,16 @@ export type GraphColor =
   | 'teal'
   | 'yellow';
 
+export type RawCounter = {|
```

Comment:
The type looks identical to `Counter` except for the type for samples, is it worth recreating it fully, or could we use generics for it? For example:

```js
export type GenericCounter<SamplesTable> = {
  ...
  samples: SamplesTable,
}

export type Counter = GenericCounter<CounterSamplesTable>;
export type RawCounter = GenericCounter<RawCounterSamplesTable>;

```

I'm not sure this would be useful because these structures are fairly small.

Reply:
Or should the Counter type move to `profile-derived`?

Reply:
> Also should the Counter type move to `profile-derived`?

Oh, definitely, I missed that.

Reply:
So, now that I've moved the type to `profile-derived`, I think I want to keep the duplication. I'd like people who want to generate their own profiles to see very straightforward type definitions in types/profile.js.